.github/workflows/release.yaml

```diff
@@ -110,6 +122,18 @@ jobs:
           file: config/.release/install.yaml
           asset_name: install.yaml
           tag: ${{ github.ref }}
+      
+      - name: Push yaml manifest into an OCI registry
+        env:
+          COSIGN_EXPERIMENTAL: 1
+        run: |
+            cosign login --username ${GITHUB_ACTOR} --password ${{ secrets.CR_PAT_ARTIFACTS }} ghcr.io
+            cd config/.release/ && \
+            flux push artifact oci://ghcr.io/${{ github.repository_owner }}/manifests/kyverno:$(git rev-parse --short HEAD) \
+            --path="." \
+            --source="$(git config --get remote.origin.url)" \
+            --revision="$(git branch --show-current)/$(git rev-parse HEAD)"
```

Comment:
Do we need the branch here ? Is this revision free form ?
How about using the commit sha ?

Reply:
Yes that's right, it should be `--revision="$(git rev-parse --short HEAD)/$(git rev-parse HEAD)"`

Reply:
I will make new suggestions once @developer-guy takes in the previous ones.